x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable.test.tsx

```diff
@@ -5,12 +5,28 @@
  */
 
 import { Subject } from 'rxjs';
-import { Embeddable } from './embeddable';
+import {
+  Embeddable,
+  LensByValueInput,
+  LensByReferenceInput,
+  LensSavedObjectAttributes,
+  LensEmbeddableInput,
+} from './embeddable';
 import { ReactExpressionRendererProps } from 'src/plugins/expressions/public';
-import { Query, TimeRange, Filter, TimefilterContract } from 'src/plugins/data/public';
+import {
+  Query,
+  TimeRange,
+  Filter,
+  TimefilterContract,
+  IndexPatternsContract,
+} from 'src/plugins/data/public';
 import { Document } from '../../persistence';
 import { dataPluginMock } from '../../../../../../src/plugins/data/public/mocks';
 import { VIS_EVENT_TO_TRIGGER } from '../../../../../../src/plugins/visualizations/public/embeddable';
+import { coreMock, httpServiceMock } from '../../../../../../src/core/public/mocks';
+import { IBasePath } from '../../../../../../src/core/public';
+import { AttributeService } from '../../../../../../src/plugins/dashboard/public';
+import { Ast } from '@kbn/interpreter/target/common';
```

Comment:
Please don't import from `target`, this is exported from `@kbn/interpreter/common`.